backend/domain/repositories/admin_repository.py

```diff
@@ -0,0 +1,15 @@
+class AdminDbRepository:
+    def __init__(self, models, email, password):
```

Comment:
Classes like BlaBlaRepository usually deal with several instances of BlaBla. In your code you need to create a new repository for each admin cause you pass admin's properties into the constructor.

More traditional way would be something like:

```python
class AdminDbRepo:
  def __init__(self, db_session):
      # ...
  def add(self, AdminInfo):
      self.session.add(,.....)
      
   def get_by_id(self, id) -> AdminInfo:
      abc = self.session.query(...)
      return AdminInfo.create_from(abc)

```